apps/web/src/lib/components/ui/community-details/AgentDetails.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+	import dayjs from 'dayjs';
+	import relativeTime from 'dayjs/plugin/relativeTime';
+	import type { Agent } from '$lib/types/models';
+
+	dayjs.extend(relativeTime);
+
+	export let AgentDisplayDetails: Agent;
+
+
+	function timeSince(dateIsoString: Date | string | number) {
```

Comment:
if I'm not mistaken, this is the new version instead of the many if statements, great stuff!! 💪 
Would love to have this in utils.ts though, so I can use this function as well :)

Reply:
Yeah, nazif told me about dayjs library , the function name inside utils.ts is the same timeSince